tool/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "excel-to-json",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha --require @babel/register"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.16.4",
```

Comment:
Do we use any of these dependencies other than xlsx-to-json?

Reply:
deleted unused ones